### PR TITLE
Game package refactor

### PR DESCRIPTION
**Pull request by pelikhan**
This PR moves the Game APIs into their own package.

- Migrate "game" apis into separate package
- Add package reference to blocksprj
- Add upgrade rules to add "game" package if missing (also applies to bluetooth, radio, devices)

TODO

- [x] ensure that microbit.org.uk scripts using "game" import properly
- [x] requires https://github.com/Microsoft/pxt/pull/892

### CODE REVIEW

File: docs/getting-started/coin-flipper.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ add these blocks to your coin flipper:
 
 #### ~
 
+Click on the ``+`` button in the toolback and add the **game** package
```
Comment (riknoll) [inline docs/getting-started/coin-flipper.md]: "toolbox"

File: docs/getting-started/rock-paper-scissors.md [hunk 2]
```diff
@@ -162,6 +162,9 @@ Here is a way you can make your Rock Paper Scissors game better.
 When button ``A`` is pressed, 
 the @boardname@ will add `1` to your score.
 
+Click on the ``+`` button in the toolback and add the **game** package
```
Comment (riknoll) [inline docs/getting-started/rock-paper-scissors.md]: "toolbox"

**Comment by abchatra** [comment]
@pelikhan Close this as well?

**Comment by pelikhan** [comment]
Yes.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Pull request closed without merging**